### ISSUE TITLE
When posting a blog post on Twitter, use Summary Card with Large Image

### ISSUE THREAD

**Issue by fliptation** [unchainedshop/unchained-website#29]
<img width="595" alt="Screenshot 2020-12-15 at 13 44 56" src="https://user-images.githubusercontent.com/600765/102216648-c2175100-3edb-11eb-9df0-ba8dd98922c1.png">

https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image

**Comment by harryadel** [comment]
My bad :smile: I just select all social medias and post. Will consider next time.

**Comment by fliptation** [comment]
@harryadel  I don't get how this is done or closed.

**Comment by harryadel** [comment]
I thought this is a heads up for future posts when using buffer, no?

**Comment by fliptation** [comment]
nope, https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image
whoever puts an unchained link into twitter should see a nice summary card

**Comment by fliptation** [comment]
@harryadel when I try to share a blog article on twitter, there is still no image shown

**Comment by Mikearaya** [comment]
@fliptation @harryadel I think this is fixed now.
